lnwallet/wallet.go

```diff
@@ -888,8 +890,8 @@ func (l *LightningWallet) handleFundingReserveRequest(req *InitFundingReserveMsg
 	req.err <- nil
 }
 
-// currentNumAnchorChans returns the current number of anchor channels the
-// wallet should be ready to fee bump if needed.
+// currentNumAnchorChans returns the current number of non-private anchor
+// channels the wallet should be ready to fee bump if needed.
```

Comment:
So this means from now on, since this will return 0 for private channels, we won't require private channels to reserver anything right?